ui-admin/src/HomePage.test.tsx

```diff
@@ -25,9 +25,9 @@ describe('HomePage', () => {
       setBreadCrumbs: jest.fn(),
       portalList,
       setPortalList: jest.fn()
-    }
+    };
 
-    ;(useNavContext as jest.Mock).mockReturnValue(mockContextValue)
```

Comment:
nit: as you know I don't have too many opinions about code style, but this is one that I _do_ have an opinion on. 🙂  I learned this convention from Nick, and I personally like keeping semi-colons at the beginning of the line when they're required. Since we don't normally use them, it's a good signal to the reviewer that using the semi-colon here is actually deliberate. And since it's this specific line of code that necessitates the use of the semi-colon in the first place, I think it makes sense to leave it here. Not a hill I'll die on, though.

Reply:
works for me -- that makes a lot of sense.  I had just assumed it was an artifact of a merge conflict rearrangement